### ISSUE TITLE
[Feature Request] Constantly on screen indication that a Begin/End Select operation has been started

### ISSUE THREAD

**Issue by alankilborn** [notepad-plus-plus/notepad-plus-plus#13047]
### Description of the Issue
The only way a user can see if the _Begin/End Select_ feature is "engaged" is by dropping the _Edit_ menu and seeing if the _Begin/End Select_ entry has a checkmark:

![image](https://user-images.githubusercontent.com/77065706/216768033-491413f1-3b6e-4f9e-b7fa-bfc1be486090.png)

It would be nice if there was some indication on another part of the UI to show when _Begin/End Select_ has "began" and is waiting to be "ended", so the user doesn't have to do the laborious step of dropping the menu down to check status ("...have I done the B/E Select or not? I can't recall...ugh, gotta drop the menu to check").

Perhaps a `B` could be added somewhere on the status bar when _Begin/End Select_  has been initiated but not yet finished?  I leave it to developers to pick the best UI representation, if they choose to implement this request.

Feature request based on current behavior in Notepad++ 8.4.9.

This issue has its origins in Community discussion.
See this [THREAD](https://community.notepad-plus-plus.org/topic/20616) and look for the `Jan 19, 2021, 4:39 PM` posting.

**Comment by alankilborn** [comment]
>Perhaps a B could be added somewhere on the status bar when Begin/End Select has been initiated but not yet finished? I leave it to developers to pick the best UI representation

Maybe the ***best*** UI representation would be a toolbar button; I didn't think of this at first because I'm not a heavy user of the toolbar buttons...

The button would show as "marked" when the first part of a two-part selection has been done; marked in this case meaning (in default theme, and non-dark mode) with a blue background.

-----------------------------------------

Thus, similar to the "indent guide" when it is "on", which appears like this:

![image](https://user-images.githubusercontent.com/77065706/216977715-d6545139-8500-464c-b8ba-08d2b1e3b950.png)

In contrast when it is "off" it appears as follows:

![image](https://user-images.githubusercontent.com/77065706/216977831-b7b287b3-9f7b-493c-af82-1e7b96cfe1cb.png)

**Comment by victorel-petrovich** [comment]
> so the user doesn't have to do the laborious step of dropping the menu down to check status

it's just `Alt+E`...

However, an indication in toolbar or status bar is probably good.

**Comment by victorel-petrovich** [comment]
I actually got here while searching if others have asked about:
 Having some marker/indicator in the text for the _position_ of Begin selection 

To make it easier to find that marker, need to also somehow mark all lines between that start position, and current position of caret: either within text, or, maybe better, in the left margin. 

@alankilborn , This then could also serve as indicator that the Begin/End Select has been started.

**Comment by alankilborn** [comment]
Yes, the logical question after knowing a begin has "begun" is _where_ did it begin.
